index.html

```diff
@@ -30,71 +30,264 @@
-              </li>
+    <main id="main">
+        <!--home page-->
+        <section class="text-headline">
+            <h1 class="primery-text">Hey There.<br />I'm Alexander</h1>
+    
+            <div class="main-description">
+            <h2 class="supporting-text">I'm a Software Developer</h2>
+    
+            <p class="description">
+                I can help you build a product, feature or website Look through some
+                of my work and experiece! If you like what you see and have a
+                project you need coded, don't hesitate to contact me.
+            </p>
+            </div>
+        </section>
+    
+        <!--icons-->
+        <ul class="icons-container">
+            <li>
+            <a href=""
+                ><img
+                src="./ressources/icons/angel.svg"
+                alt="angel"
+                class="social-icon"
+            /></a>
+            </li>
+            <li>
+            <a href=""
+                ><img
+                src="./ressources/icons/twitter.svg"
+                alt="twitter"
+                class="social-icon"
+            /></a>
+            </li>
+            <li>
+            <a href=""
+                ><img
+                src="./ressources/icons/Linkedin.svg"
+                alt="Linkedin"
+                class="social-icon"
+            /></a>
+            </li>
+            <li>
+            <a href=""
+                ><img
+                src="./ressources/icons/medium.svg"
+                alt="medium"
+                class="social-icon"
+            /></a>
+            </li>
+            <li>
+            <a href="#main"
+                ><img
+                src="./ressources/icons/github.svg"
+                alt="github"
+                class="social-icon"
+            /></a>
+            </li>
+        </ul>
+    
+        <!--illustrations-->
+        <div class="image-container"><br /></div>
+    </main>
+    
+    <!--works section-->
+    <section class="works-section">
+        <h1 class="works-section-header">My Recent Works</h1>
+        <hr />
+        <!--main grid-->
+        <div class="works-section-main-grid">
+          <div class="works-section-project">
+            <img src="./ressources/Img_Placeholder.png" alt="image-works">
+          </div>
+          <div class="works-section-project-description">
+            <h2 class="works-section-project-header">Multi-Post Stories</h2>
+            <p class="work-project-para">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+              Iusto atque possimus, repellendus recusandae dicta obcaecati minima 
+              expedita vero quidem et dolore impedit. Lorem ipsum dolor sit amet 
+              consectetur adipisicing elit. Iusto atque possimus, repellendus 
+              recusandae dicta obcaecati minima expedita vero quidem et dolore 
+              impedit. 
+            </p>
+            <ul class="language-list-body"> 
```

Comment:
please try to remove the border so they can be the same as the ones in the Figma design 

- [ ] Your design 
![image](https://user-images.githubusercontent.com/61790587/164801640-8cdecaa5-81d5-49e1-a91d-84779cc854f6.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/164801675-30ea1e3c-0f6e-44aa-a93e-cb93fecef9c4.png)